src/loaders/common/loader_helper.py

```diff
@@ -421,19 +459,19 @@ def create_hardlink_between_files(new_file, target_file):
     os.link(target_file, new_file)
 
 
-def list_objects(wsid, conf, object_type, include_metadata=False, batch_size=10000):
+def list_objects(wsid, ws, object_type, include_metadata=False, batch_size=10000):
     """
     List all objects information given a workspace ID.
```

Comment:
Should add descriptions for each arg since this is a public function

Reply:
Updated!

Reply:
: